src/components/Dialog-Window/Resources/DialogResources.module.scss

```
@@ -0,0 +1,23 @@
+.li,
+a {
+  color: rgba(31, 103, 199, 1);
+}
+
+.input {
+  width: 228px;
+  height: 32px;
+  left: 254px;
+  top: 433px;
+  border: 1px solid #eeeeee;
+  box-sizing: border-box;
+}
+
+.inputButton {
+  width: 85px;
```

Use padding instead of absolute width. `Legg til` is wider than `Add`, while the translation in some languages might be much longer and won't fit within 85px.

```suggestion
  padding: 0 1rem;
```

---

With this in mind, we should really set this on the wrapper div:

```scss
display: flex;

input[type=text] {
  flex-grow: 1;
}
```